pkg/cmd/openshift-sdn-controller/cmd.go

```diff
@@ -31,7 +32,8 @@ func NewOpenShiftNetworkControllerCommand(name string) *cobra.Command {
 			}
 		},
 	}
-
+	flags := cmd.Flags()
+	flags.BoolVar(&options.cloudDeploymment, "cloud-deployment", false, "Indicates if openshift-sdn is deployed in a public cloud")
```

Comment:
should we rather pass a type here corresponding to the infra config type? That way we know exactly what platform we are running in the context of?

Reply:
Yes, that's what I had in mind given the discussion on the Alibaba PRs. I will update this PR today and include that, then it should be ready for review.